### PR TITLE
Add variable kerning to old kern writer

### PR DESCRIPTION
**Pull request by madig**
This upgrades the old kern writer to be able to output variable kerning for when you compile features once for a VF.

### CODE REVIEW

File: Lib/ufo2ft/featureWriters/kernFeatureWriter2.py [hunk 1]
```diff
@@ -240,6 +271,136 @@ def getKerningPairs(font, side1Classes, side2Classes, glyphSet=None):
                 result.append(KerningPair(side1, side2, value))
         return result
 
+    @staticmethod
+    def getVariableKerningPairs(
```
Comment (anthrotype) [inline Lib/ufo2ft/featureWriters/kernFeatureWriter2.py:275]: better to put this somewhere it can be shared by both kern writers than copy/paste

Comment (madig) [reply Lib/ufo2ft/featureWriters/kernFeatureWriter2.py]: I'm calling into the newer writer now.

File: Lib/ufo2ft/featureWriters/kernFeatureWriter2.py [hunk 2]
```diff
@@ -208,7 +224,35 @@ def getKerningClasses(cls, font, feaFile=None, glyphSet=None):
...
+                    for glyph in glyphs.glyphSet()
+                )
+                for group_name, glyph_defs in side2Classes.items()
+            }
+            pairs = NewKernFeatureWriter.getVariableKerningPairs(
+                font,
+                side1ClassesRaw,
+                side2ClassesRaw,
+                glyphSet or {},
+                options or KernFeatureWriter.options,
```
Comment (anthrotype) [inline Lib/ufo2ft/featureWriters/kernFeatureWriter2.py:252]: options parameter is supposed to be a SimpleNamespace, but KernFeatureWriter.options class attribute is actually a dict, I suggest you convert it into a namespace before passing it onwards

**Review by anthrotype — APPROVED** [review]
LGTM with minor comment

**Comment by madig** [comment]
Ugh, the KerningPair parameters need to be converted. Why didn't I catch this...

**Pull request merged**